rllib/agents/a3c/README.md

```diff
@@ -1,22 +0,0 @@
-# Advantage Actor-Critic (A2C, A3C)
-
-## Overview 
-
-[Advantage Actor-Critic](https://arxiv.org/pdf/1602.01783.pdf) proposes two distributed model-free on-policy RL algorithms, A3C and A2C. These algorithms are distributed versions of the vanilla Policy Gradient (PG) algorithm with different distributed execution patterns. The paper suggests accelerating training via scaling data collection, i.e. introducing worker nodes, which carry copies of the central node's policy network and collect data from the environment in parallel. This data is used on each worker to compute gradients. The central node applies each of these gradients and then sends updated weights back to the workers.
```

Comment:
why remove this? is this duplicated in docs?

Reply:
Yeah, I separated A2C and A3C as discussed in our doc.